gubernator/config.yaml

```diff
@@ -29,6 +29,7 @@ jobs:
   kubernetes-jenkins/pr-logs/directory/:
   - pull-kubernetes-bazel-build
   - pull-kubernetes-bazel-test
+  - pull-kubernetes-e2e-azure
```

Comment:
this looks a lot more like it should be called pull-kuberentes-e2e-acsengine, and I'm not sure I'm ok with that

Reply:
The pattern I would prefer is `{type}-kubernetes-e2e-{provisioner}-{cloud}`, eg: `pull-kubernetes-e2e-kops-aws`.  So in this case `pull-kubernetes-e2e-aks-engine-azure`.  There are other provisioners out there that support azure, eg: kubespray

Yes, I know we have`pull-kubernetes-e2e-gce` jobs, but if renaming them wasn't so painful I would rename them to `pull-kubernetes-e2e-kube-up-gce`

Reply:
@spiffxp I've updated the name per your comment.